docs/contracts/node-operators-registry.md

```diff
@@ -116,46 +95,489 @@ function getOperatorIds()
+| `_operatorId`   | `uint256` | Id of the node operator |
 
-Returns the array of ids of all the operators that were slashed.
+### getValidator()
+Returns an instance of the validator from the stake manager
 
 ```sol
-function getIfOperatorsWereSlashed()
+  function getValidator(uint256 _operatorId)
         external
         view
+        returns (Validator memory va)
+```
+
+#### Parameters:
+
+| Name            | Type      | Description                                                         |
+| --------------  | --------- | ----------------------------------                                  |
+| `_operatorId`   | `uint256` | Id of the node operator |
+
+## Methods
+
+### addOperator()
+
+Add node operator named `_name` with reward address `_rewardAddress` and signer (heimdall) public key `_signerPubkey`
+
+```sol
+function addOperator(
+        string memory _name,
+        address _rewardAddress,
+        bytes memory _signerPubkey
+    )
+        external
+        override
+```
+
+#### Parameters:
+
+| Name             | Type      | Description                                                       |
+| ---------------- | --------- | ----------------------------------------------------------------- |
+| `_name`          | `string`  | Human-readable name                                               |
+| `_rewardAddress` | `address` | Goerli address which receives stMATIC rewards for this operator   |
+| `_signerPubKey`  | `bytes`   | Public key used on heimdall that is 64 bytes long.                |
+
+### stopOperator()
+
+Disable the node operator with given id
+
+```sol
+function stopOperator(uint256 _operatorId)
+        external
+        override
+```
+
+#### Parameters:
+
+| Name      | Type      | Description                       |
+| --------- | --------- | --------------------------------- |
+| `_id`     | `uint256` | Node Operator id                  |
+
+### exitOperator()
+
+Changes the state from WAIT to EXIT for the operator based on the given validator share address
+
+```sol
+function exitOperator(address _validatorShare) external override
+```
+
+#### Parameters:
+
+| Name      | Type      | Description                       |
+| --------- | --------- | --------------------------------- |
+| `_validatorShare`     | `address` | Address of the validator share             |
+
+### removeOperator()
+
+Removes the node operator with given id
+
+```sol
+function removeOperator(uint256 _operatorId)
+        external
         override
-        returns (bool[] memory)
 ```
 
+#### Parameters:
+
+| Name      | Type      | Description                       |
+| --------- | --------- | --------------------------------- |
+| `_id`     | `uint256` | Node Operator id                  |
+
+### joinOperator()
+
+Adds a validator that was already staked on the polygon stake manager to the PoLido protocol.
+
+```sol
+function joinOperator() external override
+```
+
+### stake()
+
+Stakes a validator on the Polygon stakeManager contract.
```

Comment:
- user have first to approve the amount  + _heimdallFee to the validatorProxy
- require operator status equal to INACTIVE